server/fly-away/src/main/java/com/codestates/flyaway/domain/board/service/BoardService.java

```diff
@@ -0,0 +1,71 @@
+package com.codestates.flyaway.domain.board.service;
+
+import com.codestates.flyaway.domain.board.entity.Board;
+import com.codestates.flyaway.domain.board.repository.BoardRepository;
+import com.codestates.flyaway.global.dto.MultiResponseDto;
+import com.codestates.flyaway.global.exception.BusinessLogicException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import static com.codestates.flyaway.global.exception.ExceptionCode.*;
+import static com.codestates.flyaway.web.board.dto.BoardDto.*;
+import static com.codestates.flyaway.web.board.dto.BoardDto.BoardResponseDto.*;
+
+@Transactional(readOnly = true)
+@Service
+@RequiredArgsConstructor
+public class BoardService {
+
+    private final BoardRepository boardRepository;
+
+    @Transactional
+    public BoardResponseDto create(Create createDto) {
+
+        Board board = new Board(createDto.getTitle(), createDto.getContent());
+        boardRepository.save(board);
+
+        return boardToBoardResponse(board);
+    }
+
+    @Transactional
+    public BoardResponseDto update(Update updateDto) {
+        long boardId = updateDto.getBoardId();
+        Board board = findById(boardId);
+        board.update(updateDto.getTitle(), updateDto.getContent());
+        boardRepository.save(board);
+
+        return boardToBoardResponse(board);
```

Comment:
```suggestion
        final Board board = boardRepository.getReferenceById(updateDto.getBoardId());
        board.update(updateDto.getTitle(), updateDto.getContent());
        return boardToBoardResponse(board);
```
`nit`
- 호불호가 있긴 하지만 jpa 는 nullsafe 조회 메서드를 제공합니다. 
- jpa 를 활용한다면 dirty check 활용을 권장합니다.

Reply:
- 관련내용을 찾아보고 학습해보겠습니다!